### ISSUE TITLE
facilitate working locally with conda

### ISSUE THREAD

**Issue by chozelinek** [interrogator/wwc#30]
@interrogator was right. At least for Windows, `Anaconda` and `conda` make things easier. Good for beginners.

We will work on it and submit a pull request, at least adapting the instructions for Windows.

@interrogator , do you have experience or know someone using Anaconda in Mac?
@alvations ,  what about Ubuntu?

**Comment by interrogator** [comment]
Hey @chozelinek , yep, I've run anaconda before, but it was ages ago. I think anaconda aims to be as cross-platform as possible, so hopefully the instructions are similar for each OS...

**Comment by chozelinek** [comment]
Great! I think for beginners (and for contributors) will simplify things, if the same instructions work cross-platform.

I just created a branch here called `anaconda`. Once we are happy with the Anaconda instructions, we can merge with master.

**Comment by alvations** [comment]
I agree with `conda` on windows, that's the only sane way to do it. We tried with pip and all on windows before (https://github.com/alvations/mt4nltk) and it was painfully unnecessary.

On Ubuntu, it seems like everyone in machine learning and data science has moved to conda. And that will simplify things too. So for now conda for Ubuntu seems sound too. Especially, when numpy/scipy is involved.

They're nice for now, they started off a little hostile when competing with pip at the start. But once they get weird again with restrictions and integration of tools, we should find alternatives.
